cylc/flow/cfgspec/suite.py

```diff
@@ -149,6 +148,9 @@
     },
     'runtime': {
         '__MANY__': {
+            'execution polling intervals': [VDR.V_INTERVAL_LIST, None],
```

Comment:
I've attempted to put these items in alphabetical order. Given the level of alphabetism in the previous version this was tricky.